src/stylus/fablevision/parent-drake.styl

```diff
@@ -33,3 +33,8 @@
     margin-top: 10px
   .organism
     margin-top: -8px
+
+.os-windows
+  .geniblocks.parentDrake
+    .parentTitle
+      margin-top: 1rem
```

Comment:
Not yet, but I haven't looked very hard. I think manually tweaking the handful we care about for the classroom test makes sense, and then we can take the time to look for a more general solution after the classroom test.